packages/aws-appsync/src/utils/logger.ts

```diff
@@ -0,0 +1,28 @@
+import debug from 'debug';
+
+export type Logger = Function & {
+    extend(category: string): Logger;
+};
+
+const noLogger: Logger = (() => {
```

Comment:
I think this wont be required (debug was set as dependency)

Reply:
I've removed it